2011-11-16 Fredy Paquet <mailbox AT opag.ch>
    * fixed include problem with glib V2.31.0 (only glib.h can be included directly)

--- gtkextra/gtkcharsel.c	2011/09/10 11:44:53	1.10
+++ gtkextra/gtkcharsel.c	2011/11/26 11:05:59	1.11
@@ -34,7 +34,7 @@
 
 
 #include <gtk/gtk.h>
-#include <glib/gunicode.h>
+#include <glib.h>
 
 #include <string.h>
 #include "gtkextra-compat.h"
